v4-client-py/v4_client_py/chain/aerial/client/utils.py

```diff
@@ -18,6 +18,7 @@ def prepare_and_broadcast_basic_transaction(
     gas_limit: Optional[int] = None,
     memo: Optional[str] = None,
     broadcast_mode: BroadcastMode = None,
+    fee: Optional[str] = "5000dv4tnt",
```

Comment:
Kind of open questions, but shouldn't fee be required? I think on the protocol it is required. Additionally on mainnet will the token be called `dv4tnt`? Not sure if we want to default to this value

Reply:
Placing orders does not require a fee.

Reply:
set the fee to be 0 if placing an order. We can change the default token when mainnet comes around?

Reply:
May make more sense to define a default `coin` based on the network in the constants [here](https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-py/v4_client_py/clients/constants.py).

Reply:
done